dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
```

Comment:
We're introducing a new package here... Is it a good place to have it under the current `api` package?